net-print/cnijfilter/cnijfilter-3.80.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+MY_P="${PN}-source-${PV}-1"
+
+DESCRIPTION="CUPS drivers for some Canon inkjet printers (common part)"
+HOMEPAGE="http://support-sg.canon-asia.com/contents/SG/EN/0100469302.html"
+SRC_URI="http://gdlp01.c-wss.com/gds/3/0100004693/01/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+net +servicetools"
+
+RDEPEND="
+	>=media-libs/libpng-1.5:0=
+	>=media-libs/tiff-3.4:0=
+	>=net-print/cups-1.4
+	servicetools? (
+		>=dev-libs/libxml2-2.7.3-r2
+		>=x11-libs/gtk+-2.6:2
+	)
+"
+DEPEND="${RDEPEND}
+	sys-devel/gettext
+"
+
+S="${WORKDIR}/${MY_P}"
+
+# Patches from bug #130645
+
+PATCHES=(
+	"${FILESDIR}/cnijfilter-3.70-png.patch"
+	"${FILESDIR}/cnijfilter-3.70-ppd.patch"
+	"${FILESDIR}/cnijfilter-3.70-ppd2.patch"
+	"${FILESDIR}/cnijfilter-3.70-libexec-cups.patch"
+	"${FILESDIR}/cnijfilter-3.70-libexec-backend.patch"
+	"${FILESDIR}/cnijfilter-3.80-cups1.6.patch"
+	)
+
+pkg_setup() {
```

Comment:
As a minor note, you could move this logic to `src_prepare` (or surround with `${MERGE_TYPE}` conditional) since I don't think it's needed when you install binary packages.